### ISSUE TITLE
Alt text

### ISSUE THREAD

**Issue by farishkash** [webgodess/prj-rev-bwfs-tea-cozy#4]
https://github.com/webgodess/prj-rev-bwfs-tea-cozy/blob/02d140c82d25968a42757352b10796ee9fe648c9/tea%20cozy/index.html#L48

All images should have descriptive alt text.  The visually impaired rely on alt text through the use of screen readers on their computers and mobile devices.
